src/analysis/convex/cone.lean

```diff
@@ -707,4 +737,93 @@ begin
   exact is_closed_Ici.preimage (by continuity),
 end
 
+lemma pointed_of_nonempty_closed
+  {K : convex_cone ℝ H} (ne : (K : set H).nonempty) (hc : is_closed (K : set H)) : K.pointed :=
+begin
+  obtain ⟨x, hx⟩ := ne,
+  let f  := λ r : ℝ, r • x,
```

Comment:
```suggestion
  let f : ℝ → H := (• x)
```
Maybe this is more readable? If you don't like it at least remove the doubled whitespace.